src/components/Tabs/Tabs.jsx

```diff
@@ -71,6 +71,16 @@ const Tab = createClass({
 		 * The content to be rendered as the `Title` of the `Tab`.
 		 */
 		Title: node,
+
+		/**
+		 * Width (with unit) to be passed as the `style` to the `Tab`
+		 */
+		width: string,
```

Comment:
Instead of having a specific prop for width, passing in a `style` prop with a `width` property seems more idiomatic. We do use specific dimension props like `width` in cases where the render function does some logic based on that width value. In this case, it's being pass thru directly to the `style` prop anyways, so might as well keep that pattern.

Reply:
I can do that. What I was going for with the `width` prop was to create an interface that declares width as an intended style-override and use-case; where a style prop is a general escape hatch and users may or may not know which styles are intended to be overridden.